includes/Modules/Sign_In_With_Google.php

```diff
@@ -421,11 +433,62 @@ private function can_render_signinwithgoogle() {
 	 */
 	private function render_button_in_wp_login_form( $content ) {
 		if ( $this->can_render_signinwithgoogle() ) {
-			$content .= '<div class="googlesitekit-sign-in-with-google__frontend-output-button"></div>';
+			ob_start();
+			/**
+			 * Display the Sign in with Google button.
+			 *
+			 * @since n.e.x.t
+			 *
+			 * @param array $args Optional arguments to customize button attributes.
+			 */
+			do_action( 'googlesitekit_render_sign_in_with_google_button' );
+			$content .= ob_get_clean();
 		}
 		return $content;
 	}
 
+		/**
+		 * Renders the Sign in with Google button markup.
+		 *
+		 * @since n.e.x.t
+		 *
+		 * @param array $args Optional arguments to customize button attributes.
+		 */
+	public function render_sign_in_with_google_button( $args = array() ) {
+		if ( ! is_array( $args ) ) {
+			$args = array();
+		}
+
+		$default_classes = array( 'googlesitekit-sign-in-with-google__frontend-output-button' );
+		$passed_classes  = array();
+		if ( ! empty( $args['class'] ) ) {
+			$passed_classes = is_array( $args['class'] ) ? $args['class'] : preg_split( '/\s+/', (string) $args['class'] );
+		}
+
+		$classes = array_map( 'sanitize_html_class', array_merge( $default_classes, $passed_classes ) );
+		$classes = array_values( array_unique( array_filter( $classes ) ) );
+
+		$attributes = array(
+			'class' => implode( ' ', $classes ),
+		);
+
+		$data_attributes = array( 'shape', 'text', 'theme' );
+		foreach ( $data_attributes as $attribute ) {
+			if ( empty( $args[ $attribute ] ) || ! is_scalar( $args[ $attribute ] ) ) {
+				continue;
+			}
+
+			$attributes[ "data-googlesitekit-siwg-{$attribute}" ] = strtolower( (string) $args[ $attribute ] );
```

Comment:
You mentioned this is here to force the data attributes to be lowercase, but it's only the _keys_ that need to be lowercase here, eg. `data-googlesitekit-siwg-{strtolower( $attribute )}`. The value can be any valid string, so this should be changed to:

```suggestion
			$attributes[ "data-googlesitekit-siwg-{strtolower( $attribute )}" ] = (string) $args[ $attribute ];
```